front/src/applications/rollingStockEditor/components/RollingStockEditorCurves.tsx

```diff
@@ -0,0 +1,134 @@
+import RollingStock2Img from 'common/RollingStockSelector/RollingStock2Img';
+import { listCurvesComfort } from 'common/RollingStockSelector/RollingStockCardDetail';
+import RollingStockCurve from 'common/RollingStockSelector/RollingStockCurves';
+import { RollingStock } from 'common/api/osrdEditoastApi';
+import { isEmpty } from 'lodash';
+import React, { Dispatch, SetStateAction, useState } from 'react';
+import Spreadsheet, { CellBase, Matrix, createEmptyMatrix } from 'react-spreadsheet';
+
+type Curve = {
+  max_efforts: number[];
+  speeds: number[];
+};
+
+export default function RollingStockEditorCurves(props: {
+  data?: RollingStock;
+  currentRs: RollingStock['effort_curves'];
```

Comment:
It is confusing, if currentRs is currentRollingStockEffortCurve, name it as such please

Reply:
Done :)